Task1/Project/Shapes/Models/Classes/ShapeBase.cs

```diff
@@ -8,8 +11,25 @@ namespace Shapes.Models.Classes
     /// </summary>
     public abstract class ShapeBase : IShape, IFileManager
     {       
+        // FIELDS
+        static IdentifierShape idOfTheShape;

```

Comment:
it is not **idOfTheShape**, it is a method that create **ID** by Type

Reply:
It has just done.